tests/unit/evidence_lm_test.py

```diff
@@ -866,270 +878,294 @@ def test_patch_off_object_logging(self):
-                "time_out",
-                f"time out not recognized/logged correctly in eval episode {i}",
-            )
-            self.assertEqual(
-                eval_stats["primary_performance"][i],
-                "correct_mlh",
-                f"time out should use mlh in eval episode {i}",
+            self.assertGreater(
+                exp.model.learning_modules[0].buffer.stats["current_mlh"][-1][
+                    "evidence"
+                ],
+                exp.model.learning_modules[0].buffer.stats["current_mlh"][6][
+                    "evidence"
+                ],
+                "evidence should have increased after moving back on the object.",
             )
 
+    def test_evidence_time_out(self):
+        pprint("...parsing experiment...")
+        config = copy.deepcopy(self.evidence_tests_time_out)
+        for c in [
+            config,
+            create_config_with_get_good_view_positioning_procedure(config),
+        ]:
+            with MontyObjectRecognitionExperiment(c) as exp:
+                pprint("...training...")
+                exp.train()
+                pprint("...check time out logging...")
+                train_stats = pd.read_csv(
+                    os.path.join(exp.output_dir, "train_stats.csv")
+                )
+                self.assertEqual(
+                    train_stats["individual_ts_performance"][0],
+                    "no_match",
+                    f"with no objects in memory individual_ts_performance"
+                    f" should be no match",
+                )
+                for i in range(5):
+                    self.assertEqual(
+                        train_stats["individual_ts_performance"][i + 1],
+                        "time_out",
+                        f"time out not recognized/logged correctly in episode {i + 1}",
+                    )
+                self.assertEqual(
+                    train_stats["primary_performance"][2],
+                    "correct_mlh",
+                    "Evidence LM should look at most likely hypothesis at time out",
+                )
+                self.assertEqual(
+                    train_stats["primary_performance"][3],
+                    "confused_mlh",
+                    "unknown object should be logged as confused_mlh at time out",
+                )
+                self.assertEqual(
+                    len(exp.model.learning_modules[0].get_all_known_object_ids()),
+                    1,
+                    "No new objects should be added to memory after time out.",
+                )
+                self.assertLessEqual(
+                    exp.model.learning_modules[0]
+                    .get_graph("new_object0", input_channel="first")
+                    .x.shape[1],
+                    32,  # max_train_steps + exploratory_steps
+                    "No new points should be added to an existing graph after time out.",  # noqa: E501
```

Comment:
We could break the string into two parts. I can't remember if you'd need to wrap it in parentheses to do it or not.

This _might_ do it.
```suggestion
                    ("No new points should be added "
                     "to an existing graph after time out."),
```

Reply:
My reasoning here and all the E501 ignores was that I indented everything for the feature flag, and I intend to remove the indents after the feature flag is removed, so I didn't want to futz around with line formatting.

Reply:
Do you feel my excuse is good enough? 😬